demo/demoTest2.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+import argparse
+import json
+
+# ./testing.py -u https://scilog.qa.psi.ch/api/v1 scilog-adminQA@psi.ch
+
+parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument("pgroup", help="Expected form: p12345")
+parser.add_argument("-u", "--url", default="http://localhost:3000", help="Server address")
+
+clargs = parser.parse_args()
+pgroup = clargs.pgroup
+url = clargs.url
+
+
+import urllib3
+from scilog import SciLog, LogbookMessage
+
+
+# urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+# connect under a given user name 
+# (likely this will be a functional account like a beamline account)
+# (note: OIDC authenticated users can not authenticate from the CLI)
+
+log = SciLog(url, options={"username": "swissfelaramis-bernina@psi.ch"})
+
+# there are two ways of defining a query expression
+
+# simple case: the where condition is built from the individual fields
+#logbooks = log.get_logbooks(ownerGroup=pgroup)
+
+# most flexible case: specify the where expression in Mongo Syntax as python dictionary
+
+logbooks = log.get_logbooks(where={"ownerGroup": pgroup},limit=10,skip=0)
+assert len(logbooks) >= 1
+logbook = logbooks[0]
+# print("Logbook:",logbook)
+
+# make sure that further actions to this logbook
```

Comment:
maybe "make sure that further actions will be pinned to this logbook"? or "will be assigned..."?

Reply:
done